internal/config/config.go

```diff
@@ -114,6 +115,8 @@ var Defaults = func(o *Config) error {
 	o.RoutingTable.RefreshInterval = 10 * time.Minute
 	o.RoutingTable.AutoRefresh = true
 	o.RoutingTable.PeerFilter = EmptyRTFilter
+
+	o.LookupCheckInterval = 5 * time.Second
```

Comment:
That is so short, is that really useful to maintain such a short cache ?
1. Why would we try to add nodes more often than this in the first place ?
2. I'm not convinced a cache here is the right thing. We should already ignore *good* peers we are connected to because they would be added to our RT. And for bad peers go-libp2p already maintain an exponential dial backoff, so connects will start to backoff if many calls to connect in short succession fails.

Reply:
You are right, I removed the cache.